interface/wx/textctrl.h

```diff
@@ -1336,6 +1351,38 @@ class wxTextCtrl : public wxControl,
     */
     virtual bool EmulateKeyPress(const wxKeyEvent& event);
 
+    /**
+        Enable or disable native spell checking on this text control if it is
+        available on the current platform.
+
+        @onlyfor{wxmsw,wxgtk}. Currently wxMSW (>= Windows 7) and wxGTK3 are
+        supported. In addition, wxMSW use requires that the text control has
+        the wxTE_RICH2 style set. wxGTK3 requires that the control has the
+        wxTE_MULTILINE style.
+
+        @param enable
```

Comment:
This needs to be updated to correspond to the actual API.

Reply:
Yes, will do that now.